psycop/common/model_training_v2/test_pipeline.py

```diff
@@ -61,7 +61,7 @@ def test_v2_train_model_pipeline(tmpdir: Path):
                 AgeFilter(min_age=4, max_age=99, age_col_name="pred_age"),
             ),
             task=BinaryClassificationTask(
-                task_pipe=BinaryClassificationPipeline(
+                pipe=BinaryClassificationPipeline(
```

Comment:
Not sure I agree with the renaming here. In my mind, `pipe` by default refers to an sklearn pipe, and we're making a distinction between the two here by calling them `task_pipe` and `sklearn_pipe`